tools/metadata.json

```diff
@@ -3,5 +3,5 @@
   "PreviewReleaseTag": "v7.0.0-rc.1",
```

Comment:
@daxian-dbw, your last commit had 3 failures in `PowerShell-CI-macos`
PSVersionTable.Verify PSCompatibleVersions has an entry for all known versions of PowerShell
```powershell
Expected collection @('1.0', '2.0', '3.0', '4.0', '5.0', '5.1', '6.0', '6.1', '6.2', '7.1') to contain '7.0', but it was not found.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/Host/PSVersionTable.Tests.ps1: line 168
168:             $version | Should -BeIn $powerShellCompatibleVersions
```
PSVersionTable.Verify PSCompatibleVersions has no unknown PowerShell entries
```powershell
Expected collection @('1.0', '2.0', '3.0', '4.0', '5.0', '5.1', '6.0', '6.1', '6.2', '7.0') to contain '7.1', but it was not found.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/Host/PSVersionTable.Tests.ps1: line 174
174:             $version | Should -BeIn $powerShellVersions
```
Requires tests.Version checks.Check for version 7.1
```powershell
Expected an exception, with type [System.Management.Automation.ScriptRequiresException] to be thrown, but no exception was thrown.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Scripting/Requires.Tests.ps1: line 90
90:             { . $File } | Should -Throw -ExceptionType ([System.Management.Automation.ScriptRequiresException])
```